Assets/SolarSystem/Planets/EarthClouds.mat

```diff
@@ -82,7 +82,7 @@ Material:
       second:
         m_Texture: {fileID: 2800000, guid: b51ecab21caa2a1458f8cbc1d379f16f, type: 3}
         m_Scale: {x: 2, y: 2}
-        m_Offset: {x: 0.04606036, y: 0}
```

Comment:
This change doesn't seem to have anything to do with your PR and is probably a side effect of Unity's editor. There are pending changes in the Phone branch that will help with Unity's editor and material modification. Please remove this file from your PR.